packages/terra-framework-site/src/app/App.jsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { HashRouter as Router, Route } from 'react-router-dom';
```

Comment:
React router [docs on HashRouter](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/HashRouter.md#hashrouter) note that HashRouter is intended to support legacy browsers, and they encourage you to configure your server to work with [BrowserHistory](https://reacttraining.com/react-router/web/api/BrowserRouter) instead.

I believe all the browsers we need to support have support for [HTML5 history API](https://developer.mozilla.org/en-US/docs/Web/API/History) which is what `<BrowserHistory>` component uses as its implementation.

Reply:
Would you believe that I spent a few days trying to get BrowserHistory working a few months back, only to run into a bunch of issues with the webpackdevserver configuration?

Would you also believe that I gave it one more shot this morning because you brought it up and got it to work?

I can believe it. I'll switch over and see what happens. I'm still not sure what would be required to use BrowserHistory on, say, a rails server, but for the site should be fine in the meantime. The type of history (browser/hash) should be configurable on an app to app basis and not impact us much at all.

Reply:
Anddddd nevermind, I'm not going to switch it (for the time being). Switching to browser history breaks the nightwatch tests and I'm going to assume the actual deployment. Since the core and clinical sites use hash history, it'd be easier to keep them in sync.

Reply:
Ok, I'm fine with using hashHistory for this, I appreciate hearing more about what work you did already trying to get browserHistory set up and working.